release/src/main/scripts/build_release_candidate.sh

```diff
@@ -380,9 +380,11 @@ if [[ $confirmation = "y" ]]; then
 
   echo "..........Copying generated javadoc into beam-site.........."
   cp -r ${GENERATE_JAVADOC} javadoc/${RELEASE}
+  cp -r ${GENERATE_JAVADOC} javadoc/current
```

Comment:
Should this clear the `current` directory first? I think this could leave pages for objects that were removed.

We also might consider just making `current` a symlink instead.

Reply:
Thanks for your help @TheNeuralBit 👍